lib/messenger-clients/SMTPClient.ts

```diff
@@ -58,7 +58,14 @@ export class SMTPClient extends MessengerClient<SMTPAccount> {
     try {
       await this.sendMessage(account, email);
     } catch (error) {
-      if (error.response) {
+      this.context.log.warn(
+        `An error occured while trying to send a message: ${JSON.stringify(
+          error,
+          null,
+          2
+        )}`
+      );
+      if (error.response?.body) {
```

Comment:
It seems that if there is no body, you do not throw the error.... 
Shouldn't you just adapt the thromw message ?

Reply:
We should indeed throw in any case there is an error with the error message 👍🏼